### ISSUE TITLE
Naming convention in strings.xml

### ISSUE THREAD

**Issue by bowiechen** [TeamAmaze/AmazeFileManager#1706]
I personally find the naming convention in [`strings.xml`](https://github.com/TeamAmaze/AmazeFileManager/blob/master/app/src/main/res/values/strings.xml) a mess and hard to follow. 

For example, 
1. In some cases, we have [`flatcase`](https://github.com/TeamAmaze/AmazeFileManager/blob/master/app/src/main/res/values/strings.xml#L42), [`snake_case`](https://github.com/TeamAmaze/AmazeFileManager/blob/master/app/src/main/res/values/strings.xml#L46), and [`camelCase`](https://github.com/TeamAmaze/AmazeFileManager/blob/master/app/src/main/res/values/strings.xml#L161). 
1. Weird string truncations like [`archtext`](https://github.com/TeamAmaze/AmazeFileManager/blob/master/app/src/main/res/values/strings.xml#L176) instead of `archive_text`. 
1. Spelling differences like `color` and [`colour`](https://github.com/TeamAmaze/AmazeFileManager/blob/master/app/src/main/res/values/strings.xml#L226). 

How do we want to reconcile the differences? It is my personal belief that snake_case is most readable.

**Comment by EmmanuelMess** [comment]
> I personally find the naming convention

I think you found something that isn't there, I don't think there is a naming convention used, there are only names.

I've always found difficult to find what info the name should contain. Should it say where the string is used? Should it have the string in the name to prevent duplication?

**Comment by bowiechen** [comment]
The name should be as descriptive, as direct, and as unambiguous as possible. For example:

- The string "Insufficient Space" should be named `insufficient_space` instead of `in_safe` (ambiguous).
- The string "Random Skin" should be named `random_skin` instead of `random` (descriptive).

**Comment by VishalNehra** [comment]
Sounds good, but considering this being an open source, we can't be too picky.. can we? 
I've also been using snake_case, so +1 from my side.

**Comment by bowiechen** [comment]
We can definitely enforce some of the coding style requirements on our end through a linter of some sort.
